app/helpers/payments_helper.rb

```diff
@@ -2,19 +2,4 @@
 
 # Helper module for Fine Payments
 module PaymentsHelper
```

Comment:
Should we just remove the helper + spec file entirely?

Reply:
Perhaps.. I left it in incase others were rebasing and there were problems. I'll remove it now.

Reply:
Pushed up a commit that removes this.